src/wallet/validatorSigner.ts

```diff
@@ -6,15 +6,19 @@ import { ValidatorPEM } from "./validatorPem";
  * Validator signer (BLS signer)
  */
 export class ValidatorSigner {
-    private secretKey: ValidatorSecretKey;
+    private readonly secretKey: ValidatorSecretKey;
 
     constructor(secretKey: ValidatorSecretKey) {
         this.secretKey = secretKey;
     }
     /**
      * Signs a message.
      */
-    async signUsingPem(pemText: string, pemIndex: number = 0, signable: Buffer | Uint8Array): Promise<Uint8Array> {
+    static async signUsingPem(
```

Comment:
Tiny breaking change in theory, but can be in minor version - since, actually, it's a "design correction change" (it's a fix).

Reply:
as discussed with @andreibancioiu, I think we can mark this as deprecated since we already have the sign method and this does not seem to bring significant benefits.